angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.component.ts

```diff
@@ -28,7 +37,7 @@ import { Component, Input } from '@angular/core';
                 <sprk-link
                   linkType="plain"
                   href="{{ item.href }}"
-                  analyticsString="{{ item.analytics }}"
+                  analyticsString="{{ item.analytics || item.analyticsString }}"
```

Comment:
Added the || condition here to support the input (prop) of analyticsString in an attempt to normalize our API across the system while still supporting the old method and avoiding a breaking change.